src/Layers/xrRenderGL/glHW.h

```diff
@@ -0,0 +1,52 @@
+#pragma once
+
+#include "Layers/xrRender/HWCaps.h"
+#include "xrCore/ModuleLookup.hpp"
+#include "SDL.h"
+#include "SDL_syswm.h"
+
+
```

Comment:
Please, remove this one empty line. Two empty lines is excessive, one is enough.